tensorflow/core/grappler/optimizers/remapper_test.cc

```diff
@@ -461,10 +461,10 @@ class RemapperFuseMatMulWithBiasTest : public RemapperTest {
 TEST_F(RemapperFuseMatMulWithBiasTest, F32) { RunTest<DT_FLOAT>(); }
 
 TEST_F(RemapperFuseMatMulWithBiasTest, Bf16) {
-#if !defined(INTEL_MKL)
+#if !defined(ENABLE_MKL)
```

Comment:
Does `bfloat16` only work with non-native mode right now? If so, please add a comment saying so. Same as other places in this file.